assets/css/style.scss

```diff
@@ -0,0 +1,49 @@
+---
+---
+
+$soft-red: #fd9e9a;
+$hard-red: #fc5834;
+$dark-gray: #595959;
+$light-gray: #9d9e9d;
+$section-headings-color: $hard-red;
+@import "{{ site.theme }}";
+
+body {
+  font-family: "Futura", "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+form {
+  input[type="submit"] {
+    background-color: $dark-gray;
+    border-radius: 5px;
+    border: none;
+    color: #fff;
+    padding: 15px 32px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    &:hover {
+      background-color: $light-gray;
+    }
+  }
+}
+
+.page-header {
+  background: $soft-red;
```

Comment:
Maybe we should use the blue from the logo for this background? It would fit the logo (so we can iterate to use elements of our logo in future website layouts) and in general blue is nicer as header (not so aggressive). What do you think? :)

Reply:
Sounds good to me! I'll give it a try